sdk/bpf/c/inc/solana_sdk.h

```diff
@@ -526,6 +526,167 @@ uint64_t sol_try_find_program_address(
     uint8_t *bump_seed
 );
 
+/**
```

Comment:
Just noticed that there is a rest test program but no c test program to exercise these changes

Reply:
OK, have to dust off the 'ol C to get a test in there..

Reply:
Looks like the C stuff needs some work from a usability perspective.  For example, if I want to sub two Bignumbers I need to pass a pointer to the `SolFfiBugNumber` struct to fill in, but that needs to be pre-allocated.  What is the code flow for the user to create one of those?